webapp/channels/src/components/youtube_video/youtube_video.tsx

```diff
@@ -113,7 +137,7 @@ export default class YoutubeVideo extends React.PureComponent<Props, State> {
                 <div>
                     {header}
                     <div
-                        className='video-div embed-responsive-item'
+                        className={cn('video-div', 'embed-responsive-item', this.state.shortsExpanded ? 'video-shorts-expanded' : (isShorts && 'video-shorts'))}
```

Comment:
Put each argument of classnames in a new line for better readability.

Also, consider using the object notation for classnames here:
```
className={classNames(
    ...,
    {
        'video-shorts-expanded': this.state.shortsExpanded,
        'video-shorts': isShorts && !this.state.shortsExpanded,
    }
)}
```